<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DivListTextEdit demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="DivListTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>380</width>
        <height>220</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionInsertOrderedList"/>
   <addaction name="ActionIncreaseIndentation"/>
  </widget>
  <action name="ActionInsertOrderedList">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/format-list-ordered.ico</normaloff>:/icons/icons/format-list-ordered.ico</iconset>
   </property>
   <property name="text">
    <string>Insert ordered list</string>
   </property>
   <property name="toolTip">
    <string>insert ordered list</string>
   </property>
  </action>
  <action name="ActionIncreaseIndentation">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/format-indent-more-5.png</normaloff>:/icons/icons/format-indent-more-5.png</iconset>
   </property>
   <property name="text">
    <string>Increase indentation</string>
   </property>
   <property name="toolTip">
    <string>increase indentation</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DivListTextEdit</class>
   <extends>QTextEdit</extends>
   <header>DivListTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
